src/pages/en/community/rules.mdx

```diff
@@ -86,9 +86,12 @@ Do not change your avatar, nickname, username or custom/rich status to try to ev
 
 #### Nicknames
 
-All nicknames must adhere to the rules above. Additionally, they should not be hard to read or too noisy. Nicknames may be considered difficult to read or noisy if they contain emojis, Zalgo or other difficult-to-read styling, or Unicode control characters. "Empty" nicknames are also considered difficult to read.
+All nicknames must adhere to the rules above. Additionally, nicknames have to adhere to the following rules:
+1. Nicknames must be easy to read, especially by a screen-reader (ex: no Zalgo or otherwise hard to read non-ascii characters, with the exception of non-Latin names)
+2. Nicknames cannot be intentionally confusing or misleading (ex: impersonation is stricly forbidden, as well as using similar looking nicknames to be intentionally confusing)
```

Comment:
typo: `stricly` -> `strictly`
Also I'd probably phrase this differently without `(ex: )`s:

```
2. Nicknames cannot be intentionally confusing or misleading — impersonation is strictly forbidden, as well as using similar-looking nicknames to intentionally confuse others.